.github/actions/minikube-setup/action.yml

```diff
@@ -0,0 +1,40 @@
+name: 'Minikube setup action'
+description: 'A wrapper around download docker artifact to allow passing a list'
+inputs:
+  image-list:
+    description: 'List of images'
+    required: true
+    default: 'kserve/agent:${{ github.sha }}'
```

Comment:
I guess we do not need the inputs here for this action

Reply:
Yes, Dan, was trying to see if we could pass a list of images but could not.
The inputs does not accept a list.
Will remove it.